objects-revisited/main.js

```diff
@@ -0,0 +1,102 @@
+const business = {
+  opens: '9AM',
+  closes: '5PM',
+  totalEmployees: 4,
+  daysOpen: ['M', 'T', 'W', 'Th', 'F'],
+  employees: {
+    Saul: {
+      position: 'Lawyer',
+      daysOfWeekWorking: ['M', 'T', 'W', 'Th', 'F'],
+    },
+    Walt: {
+      position: 'Chemist',
+      daysOfWeekWorking: ['M', 'W'],
+    },
+    Jesse: {
+      position: 'Cook',
+      daysOfWeekWorking: ['T', 'Th'],
+    },
+    Gus: {
```

Comment:
Should be lowercase for the property names. `saul`, `walt`

Reply:
Got it.